api/posts.js

```diff
@@ -25,14 +25,7 @@ router.post("/", authMiddleware, async(req, res)=>{
 
         const post = await new PostModel(newPost).save();
        
-        const postCreated = await PostModel.findById(post._id).populate("user") //so thhat we can take them and render onto front end
-        return res.json(postCreated);   //since it is populated, its easy to connect to frontend by loking at its structure

```

Comment:
Select only one way to send back post. Here I'm sending back the post without populating any data.